src/vm/amd64/unixasmhelpers.S

```diff
@@ -27,7 +27,7 @@ LEAF_ENTRY PrecodeFixupThunk, _TEXT
         lea     METHODDESC_REGISTER,[rax+r11*8]
 
         // Tail call to prestub
-        jmp     ThePreStub
+        call C_FUNC(ThePreStub)
```

Comment:
Whoops.  Fixed that while rebasing the nproc changes.  I'll include it in the squash @jkotas requested.